src/containers/nrc-content/nrc-indicators/nrc-indicators-component.jsx

```diff
@@ -60,7 +60,7 @@ function Indicators({ countryData, landMarineSelection }) {
       })}
     >
       <IndicatorCard
-        indicator={SPI ? getLocaleNumber(SPI, locale) : ''}
```

Comment:
Did we have this because we had NaN sometimes when it was loading. Maybe we can do 
`SPI || SPI === 0 ?  getLocaleNumber(SPI, locale) : ''`